database/seed.js

```diff
@@ -0,0 +1,20 @@
+const db = require('./index.js');
+
+seedGen = () => {
+  var bundleId, countDown;
+  var timerData = [];
+  for (let i = 1; i < 100; i++) {
+    bundleId = i;
+    timer = Number(Math.floor (Math.random() * 13) + 1);
+    countDown = timer;  // countdown in days, will adjust to be more granular later
+    timerData.push({
+      bundleId,
+      countDown
+    })
+  }
+  return timerData;
+};
+
+module.exports = {
+  seedGen
+}
```

Comment:
Not sure if I'm seeing the whole file but not sure if you're using the require on line 1. 

Took me a minute to understand how you're using the returned array from seedGen in navbarSave but makes sense now that I see that it's an array of objects, great job! ✓

Reply:
It's a little overkill with only 2 values, but it's a really convenient way to save the data! Came across it doing my MVP.

I have commented out line 1 for the next commit. I had originally had the seeding separated differently between the files.